apps/extension/client/src/initiateEditor.ts

```diff
@@ -27,6 +27,7 @@ type OpenLinkEvent = {
 type MachineChangedEvent = {
   type: 'MACHINE_CHANGED';
   edits: MachineEdit[];
+  options?: { v5?: boolean } | undefined;
```

Comment:
```suggestion
  options?: { xstateVersion?: 4 | 5 } | undefined;
```
Should we make it more explicit which version we want? This will also set us up for future releases.

Reply:
This will likely change in the future anyway so I'm not as concerned about it. It's also within a semi-internal package so we can always bump a major version if we need to change this